cpp/NativeOpacityTurboModule.cpp

```diff
@@ -676,9 +713,46 @@ jsi::Value NativeOpacityTurboModule::getCartaCorporationSecurities(
               rt, jsi::String::createFromUtf8(rt, err));
           reject->asObject(rt).asFunction(rt).call(rt, error);
         });
-      };
-    }).detach();
-    return {};
+      }; })
+          .detach();
+      return {};
+  }));
+  }
+
+  jsi::Value NativeOpacityTurboModule::getGithubProfile(jsi::Runtime &rt)
+  {
+    jsi::Function promiseConstructor =
+        rt.global().getPropertyAsFunction(rt, "Promise");
+        return promiseConstructor.callAsConstructor(rt, HOSTFN("promise") {
+      auto resolve = std::make_shared<jsi::Value>(rt, args[0]);
+      auto reject = std::make_shared<jsi::Value>(rt, args[1]);
+      std::thread([resolve, reject, jsInvoker = jsInvoker_, &rt]()
+                  {
+      char *json;
+      char *proof;
+      char *err;
+
+      int status = opacity_core::get_github_profile(&json, &proof, &err);
```

Comment:
Since you are doing your changes repo by repo, you need to commit the latest versions of the .a file.